inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/StringMatchingRecommender.java

```diff
@@ -126,8 +130,12 @@ public void predict(RecommenderContext aContext, CAS aCas) throws Recommendation
                     
                     // Need to check that the match actually ends at a token boundary!
                     if (tokens.stream().filter(t -> t.getEnd() == end).findAny().isPresent()) {
-                        spans.add(new Span(begin, end, text.substring(begin, end),
-                                node.value.getBestLabel()));
+                        int total = IntStream.of(node.value.counts).sum();
+                        for (LabelCount lc : node.value.getBest(maxRecommendations)) {
+                            double score = (double) lc.count / (double) total;
```

Comment:
I would describe the score you compute here.

Reply:
Done - although not here. I factored to code out into the LabelStats class and describe it there.